index.js

```diff
@@ -42,11 +42,7 @@ async function parse ({path: modulePath, files, format, importMap, output}) {
 
 function getSourceFiles (base, ignore) {
 	return new Promise((resolve, reject) => {
-		glob('**/package.json', {cwd: base}, (er, files) => {
-			if (er) {
-				reject(er);
-				return;
-			}
+		glob('**/package.json', {cwd: base}).then(files  => {
```

Comment:
Nice works. :)
Could you share the migration guide link? I cannot find it.

Reply:
Sure, here it is: https://github.com/isaacs/node-glob